src/views/common/TimeScrollView/TSVPaintAxes.ts

```diff
@@ -1,50 +1,10 @@
 import { TickSet } from "views/common/TimeScrollView/YAxisTicks";
-import { TimeScrollViewPanel, TimeTick } from "./TimeScrollView";
+import { TimeTick } from "./TimeAxisTicks";
```

Comment:
Removed about half the code in this file that dealt with different layers, and specified that this is the paint logic for the axes layer (that also handles the highlight spans), with its own name, as that's by far the most complex paint logic for the TSV.